### ISSUE TITLE
[Feature]: Snow-layer-like frames

### ISSUE THREAD

**Issue by unweeked** [XFactHD/FramedBlocks#292]
### Describe the feature

Maybe you could add a frame which is stackable in the same way snow-layers are. This would be possible in all 4 directions, essentially allowing for vertical slabs with custom thickness.
The BlockCarpentry Mod already has such a feature, maybe you could take a look at the way they do it if I didn't explain it that well.

**Comment by XFactHD** [comment]
The Framed Collapsible Cube should do what you want, it can be deformed by in any of the 6 directions (though only one at a time) and allows different deformations per corner. To move a corner inwards, hit it with a Framed Hammer, to move it outwards, hit it with a Framed Hammer while crouching.

If that doesn't allow you to do what you are looking for, then you'll need to get into more details.

**Comment by unweeked** [comment]
actually, that pretty much serves the purpose, thanks! However, is there any way to quickly reproduce certain modifications of these blocks? For example if I wanted to create a wall using a 4-pixel wide collapsible cube, would it be possible to get that without having to collapse each cube manually?

**Comment by XFactHD** [comment]
Yes, that is indeed possible: collapse one block to the desired thickness, right-click it with a Framed Blueprint while crouching to copy it (including the its camo) and then use the blueprint to place the copies. The only limitation of this is that you can't turn the copies, they will all point in the same direction as the original block.

**Comment by squigglezworth** [comment]
Would you possibly reconsider this idea? I know the collapsible cube works... but that workflow isn't exactly smooth when you're trying to place many layers of varying heights. I really think a stackable layer is worth considering.

**Comment by XFactHD** [comment]
I'll think about it. I still feel like it's duplicating a feature but I can definitely see the annoyance with the automatic height adjustment of neighboring blocks in that particular use case.

**Comment by squigglezworth** [comment]
One way of looking at it might be not duplicating a feature, but rather just making it easier to use. Collapsible Blocks will still have their uses, but this would make (what I would guess is) their most popular use-case a whole lot easier to achieve. Either way, thanks for re-opening!
